.golangci.yml

```diff
@@ -44,15 +44,18 @@ linters:
     - govet
     - ineffassign
     - interfacer
+    - makezero
```

Comment:
Why are we adding this check?
```
makezero:
    # Allow only slices initialized with a length of zero. Default is false.
```

Could you give me an example where `-predeclared` is used?
I honestly don’t understand this definition
```
  predeclared:
    # comma-separated list of predeclared identifiers to not report on
```

All checks seem to be enabled by default.
```
  thelper:
    # The following configurations enable all checks. It can be omitted because all checks are enabled by default.
    # You can enable only required checks deleting unnecessary checks.
```

Reply:
[predeclared](https://github.com/nishanths/predeclared), enabling this linter caught that the `delete` strings we had were shadowing the native go `delete` method, this linter will catch this kind of things

[makezero](https://github.com/ashanbrown/makezero), we did not have any violation for this and there's no harm in enabling it by default

[thelper](https://github.com/kulti/thelper), this is the reason I'm adding all those `t.Helper`, is a way go has to flag helper function for tests, this is useful as if we error within the helper then the line/method reported by go test is the helper method, when you call `t.Helper` then the line reported is the test that called the function